release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Text.Json.md

```diff
@@ -0,0 +1,43 @@
+# System.Text.Json
+
+``` diff
+ namespace System.Text.Json {
```

Comment:
Did we add the base serializer APIs in preview4?

cc @steveharter

Reply:
I believe we did. I would expect them to show up in this diff:
https://github.com/dotnet/corefx/blob/a9b6f23cf888a37e96ee7ffed70d487b79cbd3ab/src/System.Text.Json/ref/System.Text.Json.cs#L311-L336

Reply:
Oh nevermind, I see them now. This is per namespace, and we have them in the `3.0-preview4_System.Text.Json.Serialization.md`